routers/api/v1/repo/issue.go

```diff
@@ -130,6 +138,37 @@ func SearchIssues(ctx *context.APIContext) {
 		opts.Private = true
 		opts.AllLimited = true
 	}
+	if ctx.Form("owner") != "" {
+		owner, err := models.GetUserByName(ctx.Form("owner"))
+		if err != nil {
+			if models.IsErrUserNotExist(err) {
+				ctx.Error(http.StatusBadRequest, "Owner not found", err)
+			} else {
+				ctx.Error(http.StatusInternalServerError, "GetUserByName", err)
+			}
+			return
+		}
+		opts.OwnerID = owner.ID
+		opts.AllLimited = false
```

Comment:
why does owner affect AllLimited or AllPublic?

and tests should be extend

Reply:
```
	AllPublic       bool // Include also all public repositories of users and public organisations
	AllLimited      bool // Include also all public repositories of limited organisations
```

If those are set to true it returns those repo's, regardless of whether the owner matches or not.

I'll extend the `api_issue_test.go`